internal/config/e2eConfig.go

```diff
@@ -88,6 +88,7 @@ type KindExposePort struct {
 type Verify struct {
 	RetryStrategy VerifyRetryStrategy `yaml:"retry"`
 	Cases         []VerifyCase        `yaml:"cases"`
+	Setting       VerifySetting       `yaml:"setting"`
```

Comment:
`Setting` is not needed anymore. Move `FailFast` here.

Reply:
ok! I will change it later!